paddle/fluid/pybind/pybind.cc

```diff
@@ -777,17 +782,14 @@ All parameter, weight, gradient are variables in Paddle.
                     [](const ExecutionStrategy &self) { return self.dry_run_; },
                     [](ExecutionStrategy &self, bool dry_run) {
                       self.dry_run_ = dry_run;
-                    });
-
-  exec_strategy.def_property(
-      "use_experimental_executor",
-      [](const ExecutionStrategy &self) {
-        return self.type_ == ExecutionStrategy::kExperimental;
-      },
-      [](ExecutionStrategy &self, bool experimental) {
-        self.type_ = experimental ? ExecutionStrategy::kExperimental
-                                  : ExecutionStrategy::kDefault;
-      });
+                    })
+      .def_property(
+          "executor_type",
+          [](const ExecutionStrategy &self) { return self.type_; },
+          [](ExecutionStrategy &self, ExecutionStrategy::ExecutorType type) {
+            self.type_ = type;
+          },
+          R"DOC()DOC");
```

Comment:
Can you add more doc to describe kParallelGraph type?

Reply:
Sure, this PR is WIP, I updated the description of this PR, and add a todo list.